server/svix-server/src/v1/endpoints/attempt.rs

```diff
@@ -652,10 +652,10 @@ async fn resend_webhook(
     Extension(ref db): Extension<DatabaseConnection>,
     Extension(queue_tx): Extension<TaskQueueProducer>,
     Path((_app_id, msg_id, endp_id)): Path<(ApplicationIdOrUid, MessageIdOrUid, EndpointIdOrUid)>,
-    AuthenticatedApplication {
+    AuthenticatedOrganizationWithApplication {
```

Comment:
This should actually remain AuthenticatedApplication. Applications are allowed to send their own.